Update Readme.md (#2)
DevExpress-Examples/how-to-implement-a-theme-selector-control-similar-to-devexpress-demo-t590818: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T590818)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
<!-- default file list -->

> # ASP.NET Web Forms Controls - How to implement a Theme Selector control shown in DevExpress Demos

Reviewer: ```suggestion
# ASP.NET Web Forms Controls - How to implement the Theme Selector control shown in DevExpress Demos
```

<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/t590818/)**
<!-- run online end -->

> This example demonstrates how to implement a Theme Selector control as in [ASP.NET Web Forms Demos](https://demos.devexpress.com/ASP/). 

Reviewer: ```suggestion
This example demonstrates how to implement the Theme Selector control as shown in [ASP.NET Web Forms Demos](https://demos.devexpress.com/ASP/). 
```

![Theme Selector](image.png)

## Overview

Follow the steps below to implement a theme selector:

1. Copy files from the folders listed below to the corresponding folders in your application.

    * [App_Code](./CS/App_Code)
    * [App_Data](./CS/App_Data)
    * [Scripts](./CS/Scripts)
    * [Styles](./CS/Styles)
    * [UserControl](./CS/UserControl)

2. Register **ThemeSelector** and **ThemeParametersSelector** controls in the *web.config* file:

   ```aspx
   <pages>
     <controls>
       <!-- ... -->
       <add src="~/UserControl/ThemeParametersSelector.ascx" tagName="ThemeParametersSelector" tagPrefix="dx" />
       <add src="~/UserControl/ThemeSelector.ascx" tagName="ThemeSelector" tagPrefix="dx" />
     </controls>
   </pages>
   ```

> 3. Once the user selects a theme, the code example writes the theme to cookies. To apply this theme, handle the `Application.PreRequestHandlerExecute` event in the *Global.asax* file as follows:

Reviewer: ```suggestion
3. Once a user selects a theme, the code example writes that theme to cookies [writes to cookies???]. To apply this theme, handle the `Application.PreRequestHandlerExecute` event in the *Global.asax* file as follows:
```

Author: ```suggestion
3. Once a user selects a theme, the code example writes that theme to cookies. To apply this theme, handle the `Application.PreRequestHandlerExecute` event in the *Global.asax* file as follows:
```

* [ThemeParametersSelector.ascx.cs](./CS/UserControl/ThemeParametersSelector.ascx.cs) (VB: [ThemeParametersSelector.ascx.vb](./VB/UserControl/ThemeParametersSelector.ascx.vb))
* [ThemeSelector.ascx](./CS/UserControl/ThemeSelector.ascx) (VB: [ThemeSelector.ascx](./VB/UserControl/ThemeSelector.ascx))
* [ThemeSelector.ascx.cs](./CS/UserControl/ThemeSelector.ascx.cs) (VB: [ThemeSelector.ascx.vb](./VB/UserControl/ThemeSelector.ascx.vb))

## More Examples

> * [How to implement a Theme Selector control shown in DevExpress Demo (Old Style)](https://github.com/DevExpress-Examples/how-to-implement-a-theme-selector-control-similar-to-devexpress-demo-old-style-t504407)

Reviewer: ```suggestion
* [How to implement the Theme Selector control shown in the DevExpress Demo (Old Style)](https://github.com/DevExpress-Examples/how-to-implement-a-theme-selector-control-similar-to-devexpress-demo-old-style-t504407)
```
